metapool/prep.py

```diff
@@ -74,15 +76,33 @@ def parse_illumina_run_id(run_id):
     # this regex has two groups, the first one is the date, and the second one
     # is the machine name + suffix. This URL shows some examples
     # tinyurl.com/rmy67kw
-    matches = re.search(r'^(\d{6})_(\w*)', run_id)
+    matches6 = re.search(r'^(\d{6})_(\w*)', run_id)
 
-    if matches is None or len(matches.groups()) != 2:
+    # iSeq uses YYYYMMDD and has a trailing -XXXX value, for example
+    # 20220303_FS10001773_6_BRB11606-1914
+    # So the regex is updated to allow 8 numbers for the date, and to handle
+    # the trailing -XXXX piece
+    matches8 = re.search(r'^(\d{8})_([\w-]*)', run_id)
+
+    if matches6 is None and matches8 is None:
+        raise ValueError('Unrecognized run identifier format "%s". The '
+                         'expected format is YYMMDD_machinename_XXXX_FC.' %
```

Comment:
Looks like the error message here should be updated to also mention the format the iSeq follows.

Reply:
Ah, good spot, thank you